### PR TITLE
Add a brief page on Generative AI usage

### PR DESCRIPTION
**Pull request by willingc**
----
📚 Documentation preview 📚: https://cpython-devguide--1451.org.readthedocs.build/

### CODE REVIEW

**Review by AlexWaygood — APPROVED** [review]
Thanks, this all looks sound to me!

**Comment by diegorusso** [comment]
This is great, thanks for adding it!

File: getting-started/generative-ai.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.. _generative-ai:
+
+=============
+Generative AI
+=============
+
+Generative AI has evolved rapidly over the past decade and will continue in the future.
+Using Generative AI and LLMs (Large Language Models) can be helpful tools for contributors.
```
Comment (hugovk) [inline getting-started/generative-ai.rst:8]:
```suggestion
Using generative AI and large language models (LLMs) can be helpful tools for contributors.
```

File: getting-started/generative-ai.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Their overuse can also be problematic, such as generation of incorrect code, inaccurate documentation, and unneeded code churn.
+Discretion, good judgement, and critical thinking **must** be used when opening issues and pull requests.
+
+Acceptable uses
+===============
+
+Some of the acceptable uses of generative AI include:
+
+- Assistance with writing comments, especially in a non-native Language
```
Comment (hugovk) [inline getting-started/generative-ai.rst:17]:
```suggestion
- Assistance with writing comments, especially in a non-native language
```

File: getting-started/generative-ai.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- Gaining understanding of existing code
+- Supplementing contributor knowledge for code, tests, and documentation
+
+Unacceptable uses
+=================
+
+If a maintainer believes that a PR or issue was **fully generated** by AI tools, the PR or issue will be closed.
+If a contributor opens multiple, fully generated AI issues or PRs, the contributor may be blocked.
```
Comment (JelleZijlstra) [inline getting-started/generative-ai.rst:25]:
```suggestion
If a contributor opens multiple fully AI-generated issues or PRs, the contributor may be blocked.
```

File: getting-started/generative-ai.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+Some of the acceptable uses of generative AI include:
+
+- Assistance with writing comments, especially in a non-native Language
+- Gaining understanding of existing code
+- Supplementing contributor knowledge for code, tests, and documentation
+
+Unacceptable uses
+=================
+
+If a maintainer believes that a PR or issue was **fully generated** by AI tools, the PR or issue will be closed.
```
Comment (JelleZijlstra) [inline getting-started/generative-ai.rst:24]: I'd prefer to cast this as a more general "Issues or PRs that are not productive will be closed, including those that are AI-generated." Focus on the fact the issue is problematic, not on how the issue got there.

Comment (willingc) [reply getting-started/generative-ai.rst:24]: @JelleZijlstra Do you want to suggest wording? That makes sense to me.

File: getting-started/generative-ai.rst [hunk 5]
```diff
@@ -14,12 +14,12 @@ Acceptable uses
...
+- Assistance with writing comments, especially in a non-native language
 - Gaining understanding of existing code
 - Supplementing contributor knowledge for code, tests, and documentation
 
 Unacceptable uses
 =================
 
 If a maintainer believes that a PR or issue was **fully generated** by AI tools, the PR or issue will be closed.
-If a contributor opens multiple, fully generated AI issues or PRs, the contributor may be blocked.
+If a contributor opens multiple fully AI-generated issues or PRs, the contributor may be blocked.
```
Comment (JelleZijlstra) [inline getting-started/generative-ai.rst:25]: Maybe something like:

"Maintainers may close issues and PRs that are not useful or productive, including
those that are fully generated by AI. If a contributor repeatedly opens unproductive
issues or PRs, they may be blocked."

(Can't provide a suggestion to replace both lines at once.)

Comment (willingc) [reply getting-started/generative-ai.rst]: Adding this suggestion. Thanks!

**Pull request merged**